dashboard/client/components/Ticket/Details.tsx

```diff
@@ -0,0 +1,30 @@
+import moment from 'moment'
+import React from 'react'
+
+const TicketDetails = ({ author, createdAt, title, topics }) => {
+  const TicketDate =
+    createdAt &&
```

Comment:
createdAt is a field on every ticket so we shouldn't need to check for existence